template/app.js

```diff
@@ -29,6 +29,7 @@ import { PagesList } from './pages/PagesList'
 import { Profile } from './pages/Profile'
 import { ResetPassword } from './pages/ResetPassword'
 import { SignUp } from './pages/SignUp'
+import { MessagesList } from './pages/MessagesList'
```

Comment:
@mingxue-ordering please delete this example of theme five, this template is about template-1 (default theme), so you can't add pages of another themes, please delete it all the code about this Message pages for this example template. thanks

Reply:
all example deleted. Please check again. Thanks